### PR TITLE
Kindsys: Remove Raw kind category

### PR DESCRIPTION
**Pull request by spinillos**
It deletes raw schema generator, raw schemas (SVG) and everything related to raw files.

### CODE REVIEW

File: packages/grafana-schema/src/veneer/dashboard.types.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import * as raw from '../raw/dashboard/x/dashboard_types.gen';
+import * as raw from '../dashboard/x/dashboard_types.gen';
```
Comment (sdboyer) [inline packages/grafana-schema/src/veneer/dashboard.types.ts:1]: yeah this is what i was talking about in DM - the use of `raw` in as a path element here is independent of the raw kind category, and something we want to keep. All good, i'll fix it up :)

**Review by sdboyer — APPROVED** [review]
this looks good once all tests are passing 👍🏻

**Comment by sdboyer** [comment]
Some more context on the intent here.

With the pivot to k8s as object model target and plausibly API target, it means custom-rolling an entity API is now considerably less plausible, and at least not immediate. Along with that changes, the notion of a 'Raw' kind - basically, a not-schematized-by-Grafana type that represents particular types of file assets - is pushed further out into the future/uncertainty, and is less obviously something that should be a top-level sibling to other kinds.

Having Raw/unschematized kinds as a sibling to the other schematized kinds always made naming and and other abstractions within kindsys considerably more complex. Since it’s no longer clear that they’ll eventually be siblings, it's preferable to just pull out `Raw` completely.

Nothing should prevent us from reintroducing a `Raw` concept as a sibling later if that turns out to be right. But we've been paying a complexity tax on everything we do and try to explain in kindsys already, so it's preferable to do this now.

**Comment by sdboyer** [comment]
@ryantxu fyi - concerns?

**Comment by ryantxu** [comment]
@sdboyer no concerns -- though still trying to understand how/where the parent metadata will live.  Essentially back to the same question if these being siblings matters.

**Pull request merged**